src/modules/auth/screens/login-screen.tsx

```diff
@@ -82,33 +82,53 @@ export const LoginScreen = ({ navigation: { navigate } }: ScreenProps) => {
-          <View className='w-full mt-8'>
-            <Text variant='body1-small' className='mt-4'>{common.labels.email}</Text>
-            <TextInput autoCapitalize='none' value={email} onChangeText={setEmail} placeholder={common.place_holders.email_input}></TextInput>
-            <Text variant='body1-small' className='mt-4'>{common.labels.password}</Text>
-            <TextInput autoCapitalize='none' value={password} onChangeText={setPassword} secureTextEntry={true} placeholder={common.place_holders.password_input } ></TextInput>
+    <ImageBackground className="flex-1" resizeMode="stretch" source={images.backgroundAuth()}>
+      <View className="gap-4 p-8 pt-16">
+        <View className="items-center rounded-lg bg-white px-6 pb-4 pt-8">
+          <Image source={images.logoBlackMarket()} className="h-8 w-48"></Image>
+          <View className="mt-8 w-full">
+            <Text variant="body1-small" className="mt-4">
+              {common.labels.email}
+            </Text>
+            <TextInput
+              autoCapitalize="none"
+              value={email}
+              onChangeText={setEmail}
+              placeholder={common.place_holders.email_input}></TextInput>
+            <Text variant="body1-small" className="mt-4">
+              {common.labels.password}
+            </Text>
+            <TextInput
+              autoCapitalize="none"
+              value={password}
+              onChangeText={setPassword}
+              secureTextEntry={true}
+              placeholder={common.place_holders.password_input}></TextInput>
           </View>
           <CardNotification style={{marginTop:10,  height: 50}} />
           <Button
-          variant='primary'
-          size='large'
-          className='mt-4'
-          label='Log In'
-          onPress={() => handle()}
-           />
+            variant="primary"
+            size="large"
+            className="mt-4"
+            label="Log In"
+            onPress={() => {
+              handle();
+            }}
+          />
           <TouchableOpacity>
-            <Text className='mt-4 text-primary-800'>{common.labels.forgot_password}</Text>
+            <Text className="mt-4 text-primary-800">{common.labels.forgot_password}</Text>
           </TouchableOpacity>
         </View>
-        <View className='rounded-lg bg-white items-center px-6 py-4'>
-          <Text variant='body1'>{common.labels.without_account}</Text>
-          <Button variant='primary-inverted' size='large' className='mt-4' label={common.labels.sign_up} onPress={() => navigate("SignUp")} ></Button>
+        <View className="items-center rounded-lg bg-white px-6 py-4">
+          <Text variant="body1">{common.labels.without_account}</Text>
+          <Button
+            variant="primary-inverted"
+            size="large"
+            className="mt-4"
+            label={common.labels.sign_up}
+            onPress={() => navigate('SignUp')}></Button>
```

Comment:
This element can be self-closed
```suggestion
            onPress={() => navigate('SignUp')} />
```

Reply:
done